Add batching rule for "is_complex", "conj"

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#44649 Add batching rule for "is_complex", "conj"**

To unblock #43208, which adds "is_complex" checks to backward formulas
that are being tested for batched gradient support with vmap.

Test Plan:
- `pytest test/test_vmap.py -v`

Differential Revision: [D23685356](https://our.internmc.facebook.com/intern/diff/D23685356)

zou3519 committed:
Add batching rule for "is_complex", "conj"

To unblock #43208, which adds "is_complex" checks to backward formulas
that are being tested for batched gradient support with vmap.

Test Plan:
- `pytest test/test_vmap.py -v`

[ghstack-poisoned]

zou3519 committed:
Update on "Add batching rule for "is_complex", "conj""


To unblock #43208, which adds "is_complex" checks to backward formulas
that are being tested for batched gradient support with vmap.

Test Plan:
- `pytest test/test_vmap.py -v`

Differential Revision: [D23685356](https://our.internmc.facebook.com/intern/diff/D23685356)

[ghstack-poisoned]

anjali411 reviewed (commented):
Looks like `test_vmap` is not added in this list https://github.com/pytorch/pytorch/blob/master/test/run_test.py and that's why (despite the test failures), the CI was green for #43208

zou3519 commented:
> Looks like `test_vmap` is not added in this list https://github.com/pytorch/pytorch/blob/master/test/run_test.py and that's why (despite the test failures), the CI was green for #43208

Thanks for pointing that out. I've added it to run_test in #44656. Fingers crossed that there are no Windows or rocm issues...

zou3519 committed:
Update on "Add batching rule for "is_complex", "conj""


To unblock #43208, which adds "is_complex" checks to backward formulas
that are being tested for batched gradient support with vmap.

Test Plan:
- `pytest test/test_vmap.py -v`

Differential Revision: [D23685356](https://our.internmc.facebook.com/intern/diff/D23685356)

[ghstack-poisoned]

zou3519 committed:
Update on "Add batching rule for "is_complex", "conj""


To unblock #43208, which adds "is_complex" checks to backward formulas
that are being tested for batched gradient support with vmap.

Test Plan:
- `pytest test/test_vmap.py -v`

Differential Revision: [D23685356](https://our.internmc.facebook.com/intern/diff/D23685356)

[ghstack-poisoned]

zou3519 committed:
Update on "Add batching rule for "is_complex", "conj""


To unblock #43208, which adds "is_complex" checks to backward formulas
that are being tested for batched gradient support with vmap.

Test Plan:
- `pytest test/test_vmap.py -v`

Differential Revision: [D23685356](https://our.internmc.facebook.com/intern/diff/D23685356)

[ghstack-poisoned]
